mozilla_vpn/dashboards/mozilla_vpn.dashboard.lookml

```diff
@@ -475,7 +505,7 @@
     allow_multiple_values: true
     required: false
     ui_config:
-      type: button_group
+      type: checkboxes
```

Comment:
these filters were using button_group, which has UI problems for large numbers of values, so now they show checkboxes in a drop-down list.